common/plugins/org.jboss.tools.common/src/org/jboss/tools/common/quickfix/MarkerAnnotationInfo.java

```diff
@@ -49,101 +51,118 @@ public MarkerAnnotationInfo(List<AnnotationInfo> infos, ISourceViewer textViewer
 	}
 	
 	public List<ICompletionProposal> getCompletionProposals(AnnotationInfo info) {
-
-		if(info.isTop())
-			return getMarkerProposals(info);
-		else
-			return getProposals(info);
-		
+		ArrayList<String> proposalNames = new ArrayList<String>();
+		proposalNames.add("Surround with new element");
```

Comment:
Can we avoid hardcoding the proposal text here? We should reference to the original WTP proposal text if possible.